src/Plugin/PluginProvider.php

```diff
@@ -49,7 +49,7 @@ class PluginProvider implements Provider
     /**
      * @param Provider $provider
      * @param Plugin[] $plugins
-     * @param array    $options
+     * @param array{max_restarts: int<0, max>} $options
```

Comment:
sorry, this should be:
```suggestion
     * @param array{max_restarts?: int<0, max>} $options
```